.github/workflows/playground_examples_ci.yml

```diff
@@ -16,15 +16,7 @@
 name: Playground Examples CI
 
 on:
-  push:
-    paths:
-     - .github/workflows/playground_examples_ci_reusable.yml
-     - .github/workflows/playground_examples_ci.yml
-     - playground/backend/**
-     - playground/infrastructure/**
-     - learning/katas/**
-     - examples/**
-     - sdks/**
+  workflow_dispatch:
```

Comment:
I'm a little confused - isn't this still working fine? https://github.com/apache/beam/actions/workflows/playground_examples_ci.yml?query=branch%3Amaster

Am I looking at the wrong thing?

Reply:
As discussed offline, this is because the workflow doesn't fully exercise all functionality for all PRs/master runs, it only does that when there are potentially breaking changes.

The underlying issue here is that we're attaching to the last published sdk docker image (2.42 in this case).